new-pipeline/src/dist.js

```diff
@@ -874,9 +874,11 @@ function displayHistograms(histogramsList, dates, useTable, cumulative, trim) {
     var description = histogramsList[0].histograms[0].description
     var metric = histogramsList[0].histograms[0].measure
     var channel = $("#channel-version").val().split("/")[0]
-    var desc = getDescriptionWithLink(metric, channel, description)
-
+    var desc = getDescription(metric, channel, description)
+    var link = getDescriptionLink(metric, channel, description)
+    var div = document.getElementById('dist-caption');
```

Comment:
This is more clearly structured now, nice!

Reply:
Yeah! 😀